package.json

```diff
@@ -7,6 +7,7 @@
         "build": "nuxt build",
         "start": "nuxt start",
         "generate": "nuxt generate",
+        "preview": "nuxt preview",
```

Comment:
just curious about why adding preview command.
this might be not related to this PR.

Reply:
Yes, I accidentally add this line when I try to preview in local machine.
I'll remove it later.